dbActions.py

```diff
@@ -0,0 +1,71 @@
+import sqlite3
+import Settings
+
+
+def create_table(con):
+    """
```

Comment:
ho I just thought that some of the parameters I don't have to explain. it ok to explain again and again the same param all the time?